Platform/MeteorlakeBoardPkg/AcpiTables/EcSsdt/EcSsdt.asl

```diff
@@ -19,7 +19,7 @@ DefinitionBlock (
     External (\_SB.PC00.LPCB, DeviceObj)
     External (\_SB.HIDD, DeviceObj)
 
-    Include ("EcNvs.asl")
+    //Include ("EcNvs.asl")
```

Comment:
If EcNvs.asl is not used, remove this commented code, and remove file EcNvs.asl.

Reply:
Done